core/shared/src/main/scala/com/lynbrookrobotics/potassium/streams/Stream.scala

```diff
@@ -67,13 +105,13 @@ abstract class Stream[+T] { self =>
     */
   def relativize[O](f: (T, T) => O): Stream[O] = {
     var firstValue: Option[T] = None
-    map { v =>
+    map({ v =>
       if (firstValue.isEmpty) {
         firstValue = Some(v)
       }
 
       f(firstValue.get, v)
-    }
+    }, allowRestart = false)
   }
```

Comment:
I don't this this should be here

Reply:
Well we're basing this off of `map` to not repeat ourselves, but here this type of resulting stream can't be restarted so we want to make sure that we notify the user.